configure.ac

```diff
@@ -340,12 +346,16 @@ AM_CONDITIONAL([ENABLE_LASTLOG], [test "x$enable_lastlog" != "xno"])
 AC_SUBST([LIBSYSTEMD])
 if test "X$enable_logind" = "Xyes"; then
 	AC_CHECK_LIB([systemd], [sd_session_get_remote_host],
-		[enable_logind="yes"; [LIBSYSTEMD=-lsystemd];
-		AC_DEFINE([ENABLE_LOGIND], [1],
-			[Define to manage session support with logind.])],
-		[enable_logind="no"])
+		[
+			LIBSYSTEMD=-lsystemd
+			AC_DEFINE([ENABLE_LOGIND], [1], [Define to manage session support with logind.])
+		],[
+			AC_MSG_ERROR([libsystemd not found. If logind integration is not required, disable it using the --disable-logind option. ]dnl
+[ If logind integration is required, consider installing systemd or another package that provides libsystemd.])
```

Comment:
We should either remove space before `If` or to remove  space after `.` in previous message.
otherwise it will generate 2 spaces between sentences.

Reply:
It is intentional, two spaces between sentences are preferred by @alejandro-colomar.

Reply:
Updated